sqlalchemy_media/optionals.py

```diff
@@ -70,3 +68,22 @@ def ensure_paramiko():
 
     if paramiko is None:  # pragma: no cover
         raise OptionalPackageRequirementError('paramiko')
+
+
+# paramiko
```

Comment:
@ribrea This is not correct. It's related the previous dependency

Reply:
Sorry, i just copy the Paramiko ensure but forgot to change the comment.
Fixes #73bb6d3